### PR TITLE
Gutenframe: start Gutenlypso removal

### PR DESCRIPTION
**Pull request by vindl**
#### Changes proposed in this Pull Request

After enabling Gutenframe in production all of the Gutenlypso code
is now unreachable. In this patch we are starting the removal of the
majority of that code. The work on cleaning up the styles and remaining
packages will be handled in follow up PRs.

#### Testing instructions

1. Navigate to http://calypso.localhost:3000/block-editor/post/.
2. Make sure that Placeholder styles are loading properly.
3. Try publishing a new post.
4. Make sure that Media Library, Revisions and other iframe integrations work.
5. Try editing an existing post.
6. Test opt-in and opt-out flows.
7. Check for visual regressions.
8. Verify that Custom Post Types work as expected.

### CODE REVIEW

**Comment by rodrigoi** [comment]
Screen Shot 2019-03-12 at 14 31 10 

😍😍😍😍😍

File: client/gutenberg/editor/controller.js [hunk 1]
```diff
@@ -153,83 +53,30 @@ function getPressThisData( query ) {
...
-		context.primary = (
-			<CalypsoifyIframe
-				postId={ postId }
-				postType={ postType }
-				duplicatePostId={ duplicatePostId }
-				pressThis={ pressThis }
-			/>
-		);
+	// Set postId on state.ui.editor.postId, so components like editor revisions can read from it.
+	context.store.dispatch( { type: EDITOR_START, siteId, postId } );
```
Comment (rodrigoi) [inline client/gutenberg/editor/controller.js]: I think we don't have to dispatch this anymore.

Comment (vindl) [reply client/gutenberg/editor/controller.js]: We still need this for revision integrations. Please see https://github.com/Automattic/wp-calypso/pull/30962 for more details.

Comment (rodrigoi) [reply client/gutenberg/editor/controller.js]: 👍

**Pull request merged**